demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -85,6 +87,87 @@ def is_valid_version(self):  # type: () -> bool
         """
         return self._is_valid_version()
 
+    def collect_all_inputs_in_use(self):
+        """
+
+        Returns: list of all inputs used in playbook.
+
+        """
+        with open(self.file_path, 'r') as f:
+            playbook_text = f.read()
+        all_inputs_occurrences = re.findall(r"inputs\.[\S]*", playbook_text)
+        return [input.split('.')[1] for input in all_inputs_occurrences]
+
+    def collect_all_inputs_from_inputs_section(self):
+        """
+
+        Returns: list of all inputs from 'inputs' section of playbook.
+
+        """
+        inputs: Dict = self.current_file.get('inputs', {})
+        inputs_keys = []
+        for input in inputs:
+            key = input.get('key', '')
+            if key:
+                inputs_keys.append(key)
```

Comment:
```suggestion
            inputs_keys.append(input['key'])
```

assuming they all have `key`.